components/tyk-mdcb/templates/deployment-mdcb.yaml

```diff
@@ -81,8 +81,8 @@ spec:
           env:
             - name: TYK_MDCB_LISTENPORT
               value: "{{ .Values.mdcb.listenPort }}"
-            - name: TYK_MDCB_HEALTHCHECKPORT
-              value: "{{ .Values.mdcb.probes.healthCheckPort }}"
+            - name: TYK_MDCB_HTTPPORT
```

Comment:
I was thinking to set the environment variable based on the mdcb version tag.
Otherwise it will stop working for older MDCB version.

Reply:
I mean, if the MDCB version user has set the values file is less than 2.6, then we need to set `TYK_MDCB_HEALTHCHECKPORT`, `TYK_MDCB_HTTPPORT` otherwise.

Reply:
done!